syz-fuzzer/fuzzer.go

```diff
@@ -186,11 +186,17 @@ func main() {
 	if err != nil {
 		log.Fatalf("failed to connect to manager: %v ", err)
 	}
+	modules, err := host.CollectModulesInfo()
 	log.Logf(1, "connecting to manager...")
+	if err != nil {
```

Comment:
This also needs to be moved. And preferably move host.CollectModulesInfo() right after host.CollectMachineInfo() call. It's reasonable to have 2 calls to collect host info grouped together.